safe_transaction_service/history/tests/test_reorg_service.py

```diff
@@ -23,26 +23,19 @@
 
 
 class TestReorgService(TestCase):
-    @mock.patch.object(EthereumClient, "get_block")
+    @mock.patch.object(EthereumClient, "get_blocks")
     @mock.patch.object(
         EthereumClient, "current_block_number", new_callable=PropertyMock
     )
     def test_check_reorgs(
-        self, current_block_number_mock: PropertyMock, get_block_mock: MagicMock
+        self, current_block_number_mock: PropertyMock, get_blocks_mock: MagicMock
     ):
         reorg_service = ReorgServiceProvider()
 
         block = block_child
         block_number = block["number"]
 
-        def get_block_fn(number: int, full_transactions=False):
-            if number == block_number:
-                return block_child
-
-            if number == block_number + 1:
-                return block_parent
-
-        get_block_mock.side_effect = get_block_fn
+        get_blocks_mock.return_value = [block_child, block_parent]
```

Comment:
This is ok? get_blocks return a list of blocks, in this case we are querying 2 blocks, for example return block [1000, 1001] then the first one is the parent and the second one is the child. 
https://github.com/safe-global/safe-transaction-service/pull/1317/files#diff-b0f80c3ff20824ca54494683dd485a06d8cb04975c474db7aca1e4999ab9ef39R73

Reply:
Of course is ok! you can force a reorg :)